### PR TITLE
[ES-1325] added update identity api

### PR DESCRIPTION
**Pull request by kaifk468**

### CODE REVIEW

File: mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/Password.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package io.mosip.esignet.mock.identitysystem.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
```
Comment (balaji-alluru) [inline mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/Password.java:5]: please remove this dto, as it is no longer needed.

File: mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/IdentityData.java [hunk 2]
```diff
@@ -23,67 +21,50 @@ public class IdentityData {
...
-	
-	@Size(min = 1, message = ErrorConstants.INVALID_ZONEINFO)
+
 	String zoneInfo;
-	
-	@Size(min = 1, message = ErrorConstants.INVALID_LOCALE)
+
 	String locale;
 
+	Password password;
```
Comment (balaji-alluru) [inline mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/IdentityData.java:68]: please change the password to String.

File: mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/Password.java (hunk 1 above)
Comment (kaifk468) [reply mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/Password.java]: removed

File: mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/IdentityData.java (hunk 2 above)
Comment (kaifk468) [reply mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/IdentityData.java]: done

**Review by ase-101 — APPROVED** [review]

**Pull request merged**